_includes/v21.2/sidebar-data/manage.json

```diff
@@ -0,0 +1,517 @@
+              ]
+            },
+            {
+              "title": "GSSAPI Authentication",
+              "urls": [
+                "/${VERSION}/gssapi_authentication.html"
+              ]
+            },
+            {
+              "title": "Single Sign-on",
+              "urls": [
+                "/${VERSION}/sso.html"
+              ]
+            },
+            {
+              "title": "Rotate Security Certificates",
+              "urls": [
+                "/${VERSION}/rotate-certificates.html"
+              ]
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "title": "Monitoring and Alerting",
+      "items": [
+        {
+          "title": "Monitor Your CockroachDB Cloud Cluster",
+          "items": [
+            {
+              "title": "Cluster Overview Page",
+              "urls": [
+                "/cockroachcloud/cluster-overview-page.html"
+              ]
+            },
+            {
+              "title": "Alerts Page",
+              "urls": [
+                "/cockroachcloud/alerts-page.html"
+              ]
+            },
+            {
+              "title": "Monitoring Page",
+              "urls": [
+                "/cockroachcloud/monitoring-page.html"
+              ]
+            },
+            {
+              "title": "Statements Page",
+              "urls": [
+                "/cockroachcloud/statements-page.html"
+              ]
+            },
+            {
+              "title": "Sessions Page",
+              "urls": [
+                "/cockroachcloud/sessions-page.html"
+              ]
+            },
+            {
+              "title": "Transactions Page",
+              "urls": [
+                "/cockroachcloud/transactions-page.html"
+              ]
+            }
+          ]
+        },
+        {
+          "title": "Monitor Your CockroachDB Self-Hosted Cluster",
+          "items": [
+            {
+              "title": "Overview",
+              "urls": [
+                "/${VERSION}/monitoring-and-alerting.html"
+              ]
+            },
+            {
+              "title": "Enable the Node Map",
+              "urls": [
+                "/${VERSION}/enable-node-map.html"
+              ]
+            },
+            {
+              "title": "Use Prometheus and Alertmanager",
+              "urls": [
+                "/${VERSION}/monitor-cockroachdb-with-prometheus.html"
+              ]
+            },
+            {
+              "title": "Cluster API",
+              "urls": [
+                "/${VERSION}/cluster-api.html"
+              ]
+            },
+            {
+              "title": "Third-Party Monitoring Integrations",
+              "items": [
+                {
+                  "title": "Overview",
+                  "urls": [
+                    "/${VERSION}/third-party-monitoring-tools.html"
+                  ]
+                },
+                {
+                  "title": "Datadog",
```

Comment:
I think these titles are too terse. Monitoring with Datadog? Same for Kibana.

Reply:
Fixed.